Sources/StreamChat/Database/DTOs/ChannelDTO.swift

```diff
@@ -7,7 +7,11 @@ import Foundation
 
 @objc(ChannelDTO)
 class ChannelDTO: NSManagedObject {
-    @NSManaged var cid: String
+    @NSManaged private var cid: String
+    var channelId: ChannelId {
+        try! .init(cid: cid)
```

Comment:
Do we really need this? ChannelId is our decoded value, IMHO it's better to keep it the way it was (using `cid` and `channelId.rawValue`)